### ISSUE TITLE
v1.0 additional credits

### ISSUE THREAD

**Issue by brunobowden** [WorldHealthOrganization/app#883]
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

Updated "Thanks to:" section

Credits to add:
* Biswa Gurubacharya
* Bernardo Mariano Junior
* Evan Pye
* Flavio Aggio
* Andy Pattison
* Michelle Thulkanam
* Tedros Adhanom Ghebreyesus

**Comment by advayDev1** [comment]
* Michael Nichols

**Comment by advayDev1** [comment]
Security team!
Will Ackerly
Jobert Abma
Michiel Prins
Peter Nancarrow
Ryan Speers
Sitora Serverova

**Comment by b099l3** [comment]
This looks like it is just updating this [file](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) with the names listed above.

Also there are theses todos which may include other people:
# TODO: Pre-launch, review above list to add in the full team that doesn't make git commits!
# TODO: Add businesses supporting us with donated resources.

Can I work on this as a first issue?

**Comment by advayDev1** [comment]
needs to be updated with list of WHE collaborators too

**Comment by tabassumwajid2005** [comment]
> others not included above or in the list already
> 
> Security:
> Will Ackerly
> Jobert Abma
> Michiel Prins
> Peter Nancarrow
> Ryan Speers
> Sitora Serverova
> Marc McLaughlin
> Andrew Nebus
> 
> PgM:
> Marie Debard
> Stephanie Liu
> 
> PM:
> Jessica Ladd
> 
> QA:
> Mohit Saini
> Tabassum Wajid

> Legal:
> Eve Chaurand
> Celine Guiillou

**Comment by tabassumwajid2005** [comment]
Hi Bruno,

I have worked as a QA and contribute my efforts in this but it is the
second update of list and I am still not in the list. Is there any reason
of it or just a mistake?

Please add my name in the list of contributors and I am one of the
contributor of this effort.

Regards,
Tabassum wajid

**Comment by brunobowden** [comment]
@tabassumwajid2005 - my sincere apologies. Part of the reason to put this up was to ask people to check if anyone was missing. So I'm glad you caught this. Please see this PR where in fact your name gets a very special place right next to Dr Tedros himself:

https://github.com/WorldHealthOrganization/app/pull/1691/files

I was going to leave open the PR for now incase others name come along. It's tagged with "v1 - Nigeria" which means I assure you it will be included in the public release. Talking of which, we're getting close to release, so if you're able to help out on the QA, it would be much appreciated.

**Comment by tabassumwajid2005** [comment]
Thanks a lot Bruno

**Comment by tabassumwajid2005** [comment]
Yes I have installed the latest build and playing around it will share my
findings and suggestions soon.

Regards,
Tabassum
